web/source/build.sh

```diff
@@ -255,12 +244,14 @@ while [[ $# -gt 0 ]] ; do
     case $key in
         -ui)
             set_default_vars
+            BUILD_LMLAYER=false
             BUILD_EMBED=false
             BUILD_FULLWEB=false
             BUILD_COREWEB=false
             ;;
         -test)
             set_default_vars
+            BUILD_LMLAYER=false
```

Comment:
Made update the usage (ll. 9-10) to note that `-ui` and -test` won't build lmlayer.